types/jest/jest-tests.ts

```diff
@@ -239,6 +239,17 @@ describe('Assymetric matchers', () => {
     });
 });
 
+describe('setTimeout', () => {
+    it('works as expected', done => {
+        jest.setTimeout(1000);
+
+        setTimeout(() => {
+                expect(true).toBeTruthy();
```

Comment:
Thanks for adding the test. Can we change the indentation to 4 spaces in this block for consistency?